TrackletAlgorithm/TrackletLUTs.h

```diff
@@ -0,0 +1,185 @@
+#ifndef TrackletAlgorithm_TrackletLUTs_h
+#define TrackletAlgorithm_TrackletLUTs_h
+
+#include "Constants.h"
+
+template<TF::seed Seed>
+class InvdrLUT{
+
+ public: 
+
+  InvdrLUT() {
+
+    if (Seed <= TF::L5L6) {
+      for(int idr=-256; idr<256; idr++){
+	int uidr = idr;
+	if (uidr < 0) uidr += 512;
+	int idrabs = idr;
+	if (Seed == TF::L1L2 ) {
+	  idrabs += (rmean[1] - rmean[0]);
+	}
+	if (Seed == TF::L2L3 ) {
+	  idrabs += (rmean[2] - rmean[1]);
+	}
+	if (Seed == TF::L3L4 ) {
+	  idrabs += (rmean[3] - rmean[2]);
+	}
+	if (Seed == TF::L5L6 ) {
+	  idrabs += (rmean[5] - rmean[4]);
+	}
```

Comment:
@aryd Could this be simplified by using `seedLayers` in Constants.h?:
```suggestion
	idrabs += (rmean[seedLayers[Seed][1]] - rmean[seedLayers[Seed][0]]);
```

Reply:
Yes - code has been simplified.